skyportal/models.py

```diff
@@ -2517,6 +2519,12 @@ class Spectrum(Base):
         doc="Median UTC ISO time stamp of the exposure or exposures in which the Spectrum was acquired.",
     )
     origin = sa.Column(sa.String, nullable=True, doc="Origin of the spectrum.")
+    type = sa.Column(
+        allowed_spectrum_types,
+        nullable=False,
+        default=default_spectrum_type,
+        doc="Type of spectrum: source, host, etc.",
```

Comment:
Must be one of `ALLOWED_SPECTRUM_TYPES`, not etc, no?

Reply:
yes, correct.